packages/block-library/src/embed/index.js

```diff
@@ -412,8 +414,12 @@ function getEmbedBlockSettings( { title, description, icon, category = 'embed',
 				const preview = url && getEmbedPreview( url );
 				const previewIsFallback = url && isPreviewEmbedFallback( url );
 				const fetching = undefined !== url && isRequestingEmbedPreview( url );
+				// Some WordPress URLs that can't be embedded will cause the API to return
```

Comment:
Should this be normalized at the REST API / data store, rather than delegating the responsibility to the consuming UI component to handle ?